packages/manager/modules/account-sidebar/src/controller.js

```diff
@@ -28,15 +21,14 @@ export default class OvhManagerAccountSidebarCtrl {
   }
 
   $onInit() {
-    return this.$q
-      .when(this.me ? this.me : this.OvhApiMe.v6().get().$promise)
-      .then((me) => {
-        this.me = me;
-        this.hasChatbot = constants.CHATBOT_SUBSIDIARIES.includes(
-          me.ovhSubsidiary,
-        );
-      })
-      .then(() => this.$translate.refresh())
+    if (!this.me) {
```

Comment:
i believe you can remove all the ```if(!this.me)```test since you get user from environment now

Reply:
As it's a component binding (https://github.com/ovh/manager/blob/master/packages/manager/modules/account-sidebar/src/component.js#L9), so I prefer to keep it.
If `me` is defined we use it, else we use the env user.

IMHO, I think we can refactor it later.